### ISSUE TITLE
Prototype Design on Figma

### ISSUE THREAD

**Issue by jiin-kim109** [sfuSwSo/swe-resume-evaluator#8]
### MainPage and ReportingPage  
![Image](https://github.com/sfuSwSo/swe-resume-evaluator/assets/57239105/b126ed0e-2d3d-4e2c-af62-7b06c5ffb379)

![Image](https://github.com/sfuSwSo/swe-resume-evaluator/assets/57239105/de7f84bb-3695-4f47-b0f6-5c4595fbeece)  

### Drag and Drop Indicator  
(covers the entire screen)  
![Image](https://github.com/sfuSwSo/swe-resume-evaluator/assets/57239105/1458a5d8-144c-4247-b157-11102d43c2ad)

**Comment by GripenANM** [comment]
For the progress, I suggest that we do a full screen overlay rather than having the guidelines at the bottom

**Comment by jiin-kim109** [comment]
@GripenANM ticket updated. For now let's have the guidelines under the dropzone. The reporting page will be a compact full screen overlay as we discussed. We can discuss about having both in full screen overlays later. Also, having the sidemenu vs footer hasn't been decided, so you can leave it as a simple hamburger button in the header and a draft footer.   

Also, you can use the UI libs' components on Figma but for actual implementation, we won't be using them. Because the implementations are not powered by UI libs, we'd be better have a minimal design rather than a complex design, with one or two primary colours.

**Comment by GripenANM** [comment]
@jiin-kim109 Instead of guidelines at the bottom, it will be better to have it in a multi step “form” where it first shows if the user accepts the T&C, than photo of the upload document requirements, than finally the upload zone.

**Comment by jiin-kim109** [comment]
@GripenANM yes but that will also increase the workload than simple upload button with the guidlines. The workload itself isn’t a problem but this has to be discussed with all members in our retrospective meeting. You don’t have to think of a complete prototype yet.
